installation/install-by-rpm.md

```diff
@@ -70,7 +70,7 @@ $ sudo systemctl status td-agent.service
 ● td-agent.service - td-agent: Fluentd based data collector for Treasure Data
    Loaded: loaded (/lib/systemd/system/td-agent.service; disabled; vendor preset: enabled)
    Active: active (running) since Thu 2017-12-07 15:12:27 PST; 6min ago
-     Docs: https://docs.treasuredata.com/articles/td-agent
+     Docs: https://docs.treasuredata.com/display/public/PD/Installing+td-agent+on+RHEL+and+CentOS
```

Comment:
Ditto

```
● td-agent.service - td-agent: Fluentd based data collector for Treasure Data
   Loaded: loaded (/usr/lib/systemd/system/td-agent.service; enabled; vendor preset: disabled)
   Active: active (running) since Tue 2022-09-27 06:14:36 UTC; 1s ago
     Docs: https://docs.treasuredata.com/display/public/PD/About+Treasure+Data%27s+Server-Side+Agent
  Process: 33953 ExecStart=/opt/td-agent/bin/fluentd --log $TD_AGENT_LOG_FILE --daemon /var/run/td-agent/td-agent.pid $TD_AGENT_OPTIONS (code=exited, status=0>
 Main PID: 33959 (fluentd)
    Tasks: 10 (limit: 4958)
   Memory: 78.1M
   CGroup: /system.slice/td-agent.service
           ├─33959 /opt/td-agent/bin/ruby /opt/td-agent/bin/fluentd --log /var/log/td-agent/td-agent.log --daemon /var/run/td-agent/td-agent.pid
           └─33962 /opt/td-agent/bin/ruby -Eascii-8bit:ascii-8bit /opt/td-agent/bin/fluentd --log /var/log/td-agent/td-agent.log --daemon /var/run/td-agent/td>
```